lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -660,6 +660,16 @@ def set_severity(record, entry)
       elsif record.key?('severity')
         entry.metadata.severity = parse_severity(record['severity'])
         record.delete('severity')
+      elsif @service_name == CONTAINER_SERVICE && \
+            entry.metadata.labels.key?("#{CONTAINER_SERVICE}/stream")
+        stream = entry.metadata.labels["#{CONTAINER_SERVICE}/stream"]
+        if stream == 'stdout'
+          entry.metadata.severity = 'INFO'
+        elsif stream == 'stderr'
+          entry.metadata.severity = 'ERROR'
+        else
+          entry.metadata.severity = 'DEFAULT'
```

Comment:
It's not expected to ever happen, but having DEFAULT be the fail-safe default value seems pretty reasonable.